config/locales/views/admin/en.sidenav_billing.yml

```diff
@@ -0,0 +1,15 @@
+en:
+  views:
+    admin:
+      shared:
+        sidenav_billing:
+          transactions_header: Transactions
+          all_transactions: All Transactions
+          move_transactions: "!facilities.movable_transactions.head!"
```

Comment:
What does this do?

Reply:
It's a feature of [text_helpers](https://github.com/ahorner/text-helpers#suggested-use) that allows you to reference other locale entries. I like to use it sometimes in menus to reference the page title it points to so we're not having the string "Move Transactions" in two places.